src/app/map-tool/feature-overview/feature-overview.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="modal-header">
+  <app-ui-close-button class="close" (onPress)="onCancelClick($event)"></app-ui-close-button>
+</div>
+<div class="modal-body">
+  <ngx-carousel [inputs]="carousel">
+    <ngx-item NgxCarouselItem>
+      <div class="slide-animation">
+        <img src="./assets/images/all-shapes.svg" />
```

Comment:
These will need to source the `deployUrl` environment variable to work on the deployed sites